test/e2e/pkg/testnet.go

```diff
@@ -388,7 +367,7 @@ func (n Node) Validate(testnet Testnet) error {
 
 	}
 	switch n.Mempool {
-	case "", config.MempoolV0, config.MempoolV1, config.MempoolV2:
+	case "", "v0", "v1":
```

Comment:
[question] why is this missing v2?

```suggestion
	case "", "v0", "v1", "v2":
```

Reply:
you might have been reviewing an older commit https://github.com/celestiaorg/celestia-core/pull/1609/commits/677be4a6c95c857778bb6c68a69d00c6af742d2b

good catch though